lessons/Data-Science/bar-and-pie-charts/langs/en-us/index.adoc

```diff
@@ -74,20 +74,20 @@ Turn to @printable-exercise{bar-and-pie-notice.adoc} and take two minutes to not
 
 @image{images/count-more-species.png, 160} @image{images/bar-more-species.png, 385}
 
-Both `bar-chart` and `count` *group the data in one column by value*, creating "buckets" for each unique value and counting the number of rows in each bucket, one uses a picture and one uses a table.
+Both `bar-chart` and `count` *group the data in one column by value*, creating "buckets" showing the frequency of each unique value, one uses a picture and one uses a table.
```

Comment:
This is a run on (and also used to be a run on!) please add a period or semicolon before "one uses a picture"